EmpleoDotNet/Views/Home/Index.cshtml

```diff
@@ -16,7 +16,7 @@
                 <div class="jobs">
                     @foreach (var jobOpportunity in Model)
                     {
-                        <a href="@Url.Action("Detail", "JobOpportunity", new {id = jobOpportunity.Id})" class="featured" target="_blank">
```

Comment:
Eso se implemento en un PR, pero a decir verdad tienes razon, si estoy viendo trabajos para que quiero verlo en otro tab, es tan simple como click derecho open in new tab